src/components/BreedsList.vue

```diff
@@ -129,18 +131,35 @@ export default {
     scrollToTop() {
       this.$refs.resultListContainer.scrollTo({ top: 0, behavior: "smooth" });
     },
+    toggleCard(breed) {
+      breed.isActive = !breed.isActive;
+      this.displayedBreeds.forEach((item) => {
+        if (item.id !== breed.id) item.isActive = false;
+      });
+    },
```

Comment:
Tako sem implementiral, upam da bo ok :)

Reply:
je uredu.. lahko bi pa obrnil in najprej vsem nastavu na false, pol pa samo temu na true :D ampak je uredu 👍 🚀